src/helpers/utils.ts

```diff
@@ -159,3 +156,41 @@ export const createMonthsFromKeys = (startYear: number, endYear: number) => {
     })
     .flat();
 };
+
+export const removeDuplicateSummaries = (groups: Record<string, Array<HomepageEntryData>>) => {
+  const finalValues: HomepageEntryData[] = [];
+  const cleanedGroups: Record<string, Array<HomepageEntryData>> = {};
+
+  for (const [key, value] of Object.entries(groups)) {
+    if (value.length === 1 || key.toLowerCase().startsWith("combined summary")) {
+      cleanedGroups[key] = value;
+    }
+  }
+
+  const singleSummaries = Object.keys(cleanedGroups)
+    .filter((post) => !post.toLowerCase().startsWith("combined summary"))
+    .map((x) => x.toLowerCase());
+
+  const combinedSummaries = Object.keys(cleanedGroups)
+    .filter((post) => post.toLowerCase().startsWith("combined summary"))
+    .map((x) => x.toLowerCase());
+
```

Comment:
you might want to add a comment here, as it gets tricky of what exactly the code does until detailed inspect